extensions/tokenizers/src/main/java/ai/djl/huggingface/tokenizers/HuggingFaceTokenizer.java

```diff
@@ -305,60 +306,98 @@ public String decode(long[] ids) {
     /**
      * Sets the truncation and padding behavior for the tokenizer.
      *
-     * @param truncationStrategy the {@code TruncationStrategy} to use
-     * @param paddingStrategy the {@code PaddingStrategy} to use
-     * @param maxLength the maximum length to pad/truncate sequences to
+     * @param truncation whether to enable truncation
+     * @param padding whether to enable padding
+     * @param maxLength maximum length of sequences to truncate and/or pad to
      */
-    public void setTruncationAndPadding(
-            TruncationStrategy truncationStrategy,
-            PaddingStrategy paddingStrategy,
-            long maxLength) {
-        setTruncationAndPadding(truncationStrategy, paddingStrategy, maxLength, 0, 0);
+    public void setTruncationAndPadding(boolean truncation, boolean padding, long maxLength) {
```

Comment:
This looks like you are really trying to have two separate actions for truncation and padding here. Maybe split it into two functions?

Reply:
I think keeping it as a single method is more in line with the interface exposed by transformers for fast tokenizers https://huggingface.co/docs/transformers/v4.20.1/en/main_classes/tokenizer#transformers.PreTrainedTokenizerFast.set_truncation_and_padding. 

Though, based on which implementation the user chooses they can modify truncation and padding independently. Here's the rough interface provided at each level in hugging face:
- Transformers FastTokenizer: 1 method to setTruncationAndPadding https://huggingface.co/docs/transformers/v4.20.1/en/main_classes/tokenizer#transformers.PreTrainedTokenizerFast.set_truncation_and_padding - i think the primary motivation for this is that since truncation and padding can share some settings (max_length), making sure they are set together prevents the user from making mistakes
- Tokenizer python bindings: independent methods for truncation and padding https://github.com/huggingface/tokenizers/blob/main/bindings/python/py_src/tokenizers/implementations/base_tokenizer.py
- Tokenizer Rust: independent methods for truncation and padding https://github.com/huggingface/tokenizers/blob/main/bindings/python/src/tokenizer.rs